lib/intercepted_request_router.js

```diff
@@ -155,51 +175,45 @@ class InterceptedRequestRouter {
       req.once('finish', callback)
     }
 
-    if (!req.aborted && !this.playbackStarted) {
-      req.write(chunk, encoding)
-      this.startPlayback()
-    }
-    if (req.aborted) {
-      this.emitError(new Error('Request aborted'))
-    }
+    req.write(chunk, encoding)
+    req.finished = true
+    this.maybeStartPlayback()
 
     return req
   }
 
   handleFlushHeaders() {
     debug('req.flushHeaders')
-    const { req } = this
-
-    if (!req.aborted && !this.playbackStarted) {
-      this.startPlayback()
-    }
-    if (req.aborted) {
-      this.emitError(new Error('Request aborted'))
-    }
+    this.maybeStartPlayback()
   }
 
   handleAbort() {
     debug('req.abort')
-    const { req, response, socket } = this
+    const { req, socket } = this
 
     if (req.aborted) {
       return
     }
-    req.aborted = Date.now()
-    if (!this.playbackStarted) {
-      this.startPlayback()
+    req.aborted = true // as of v11 this is a bool instead of a timestamp
```

Comment:
What does the v11 in this comment refer to? Node 11?

Reply:
yeah, that was a left over note to myself. I clarified in the commit/PR desc.
I can either clarify it here or remove this comment all together.

Reply:
Either way. I think as much as possible the code comments should explain the behavior and the implementation of the code rather than having to do forensics (especially on PRs). Although explaining this specific point doesn't seem essential.